fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java

```diff
@@ -152,7 +152,7 @@ public BrokerScanNode(PlanNodeId id, TupleDescriptor destTupleDesc, String planN
         this.fileStatusesList = fileStatusesList;
         this.filesAdded = filesAdded;
         if (ConnectContext.get() != null) {
-            this.userIdentity = ConnectContext.get().getUserIdentity();
```

Comment:
This has been changed in master branch. May be you need to rebase the code

Reply:
OK,  I will close it and create new PR:  this https://github.com/apache/incubator-doris/pull/9984